examples/AspNet/Global.asax.cs

```diff
@@ -14,8 +14,9 @@ public class WebApiApplication : HttpApplication
         protected void Application_Start()
         {
             this.openTelemetry = OpenTelemetrySdk.CreateTracerProvider(
-                (builder) => builder.AddDependencyInstrumentation()
-                .AddRequestInstrumentation()
+                 (builder) => builder
+                 .AddHttpInstrumentation()
+                 .AddRequestInstrumentation()
```

Comment:
separately -should we rename to AddAspNetInstrumentation? Its more clear than AddRequestInstrumentation.

Reply:
I can change right now. no problem.